src/test/java/de/adesso/kicker/user/UserControllerTest.java

```diff
@@ -44,6 +45,9 @@
     @MockBean(name = "notificationService")
     private NotificationService notificationService;
 
+    @MockBean(name = "trackedStatisticsService")
+    private TrackedStatisticService trackedStatisticService;
```

Comment:
Can you add a test to make sure the data is loaded correctly and the template renders?

Reply:
Done